app/templates/new/base.html

```diff
@@ -6,6 +6,7 @@
 
 <!DOCTYPE html>
 <html lang="{{ htmlLang | default('en') }}" class="{{ htmlClasses }}">
+  {# {% include "new/components/head.html" %} #}
```

Comment:
This is being hidden or not included for now?

Reply:
Correct because we're not using the template yet - we'll test all this when we're ready to make the switch.